### PR TITLE
Update documentation

### LINKED ISSUE #2: Update Documentation
- Update User Guides
- Update Developer Guides
- Update README and AboutMe documents

### PR DESCRIPTION
**Pull request by AhQuanz**

### CODE REVIEW

File: docs/UserGuide.md
Comment (daniellau88) [inline docs/UserGuide.md]: For this portion, I think would be better to use the one in @weixue123 's branch, so might need to merge it in first.

[refer to this portion](https://github.com/AY2021S2-CS2103-W16-2/tp/compare/master...weixue123:inc/doc#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR146)

File: docs/DeveloperGuide.md
Comment (daniellau88) [inline docs/DeveloperGuide.md]: Would be nice to have spacing between CHIM and the full stop

File: docs/UserGuide.md
Comment (daniellau88) [inline docs/UserGuide.md]: Are we keeping this portion? Can refer below for the modified command in @weixue123 's repo

[code portion](https://github.com/AY2021S2-CS2103-W16-2/tp/compare/master...weixue123:inc/doc#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR28)

File: docs/UserGuide.md
Comment (AhQuanz) [reply docs/UserGuide.md]: Sure. Merge his PR and then i will resolve the conflict and add in the missing actions

File: docs/DeveloperGuide.md
Comment (AhQuanz) [reply docs/DeveloperGuide.md]: Noted.

File: docs/UserGuide.md
Comment (AhQuanz) [reply docs/UserGuide.md]: Yeap , will merge his changes into my branch after his PR has been merged

**Comment by aaronsms** [comment]
I will recommend squashing your commits before merging if possible.

**Review by aaronsms — APPROVED** [review]
Okay looks good.

**Pull request merged**
